Automattic/jetpack
# Repo Gardening: Prevent CRM release dates showing on PR github-actions bot comments

## Proposed changes:

* This PR prevents 'next scheduled release' and 'scheduled code freeze' dates from showing in the 'github-actions' bot comment that displays once any new PR is created, if the plugin it affects is CRM.
* The reason for this is that there is currently no scheduled date for the next release and this is often the case - though more relevant at the moment (we could always change this back 6 months or so down the line if more relevant then).
* This is to prevent confusion for any external contributors, or internal contributors from other teams. Example: https://github.com/Automattic/jetpack/pull/33712#issuecomment-1773501318

### Other information:


## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test this, you should ideally be able to create a fork of the Jetpack repository and merge this PR there. Then create a PR touching the CRM plugin, and the github-actions bot comment should not show the CRM plugin release dates ~(in my fork I am encountering a repo-gardening error though, both with and without the change present - `Error: Module parse failed: Identifier 'crmModules' has already been declared (205:8)` relating to line `const crmModules = file.match( /^projects\/plugins\/crm\/modules\/(?<crmModule>[^/]*)\// );`.~ - now fixed, thanks for the tip @tbradsha
